src/js/EditsContainer.jsx

```diff
@@ -0,0 +1,128 @@
+var React = require('react');
+var EditsSyntacticalValidity = require('./EditsSyntacticalValidity.jsx');
+
+var EditsContainer = React.createClass({
+  getInitialState: function() {
+    return {
+    	"edits": {
+    		"syntactical": [],
+    		"validity": [],
+    		"quality": [],
+    		"macro": []
+    	}
+    }
+  },
+
+  componentDidMount: function() {
```

Comment:
`componentDidMount` should be `componentWillMount`, otherwise you'll be waiting on the first render before setting the initial data